src/driver/amdxdna/amdxdna_devel.c

```diff
@@ -21,12 +21,12 @@ MODULE_PARM_DESC(iommu_mode, "0 = w/ PASID (Default), 1 = wo/ PASID, 2 = Bypass"
 
 int amdxdna_iommu_mode_setup(struct amdxdna_dev *xdna)
 {
+	struct iommu_domain *domain = {};
```

Comment:
Why it can't be initialized by NULL?

Reply:
We have not used this initialization method. Could you follow the existing coding style and initialize it to NULL?